Decrypt_PDF/Decrypt_PDF/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Syncfusion.Compression.Base" version="20.3.0.59" targetFramework="net472" />
-  <package id="Syncfusion.Licensing" version="20.3.0.59" targetFramework="net472" />
-  <package id="Syncfusion.Pdf.AspNet" version="20.3.0.59" targetFramework="net472" />
+  <package id="Syncfusion.Compression.Base" version="27.1.53" targetFramework="net462" />
+  <package id="Syncfusion.Licensing" version="27.1.53" targetFramework="net462" />
+  <package id="Syncfusion.Pdf.AspNet" version="27.1.53" targetFramework="net462" />
```

Comment:
Don't use deprecated package, instead of using winforms package?

Reply:
added